intelmq/tests/bots/experts/abusix/test_expert.py

```diff
@@ -33,7 +33,6 @@
 
 
 @test.skip_internet()
-@test.skip_travis()
```

Comment:
These tests failed regularly in the past and to not cause confusion for pull requestors for failings tests, we disabled the flaky tests in the past.

Reply:
Fixed